manifests/profiles/dualstack.yaml

```diff
@@ -0,0 +1,179 @@
+apiVersion: install.istio.io/v1alpha1
```

Comment:
can we keep the profile to just the minimum set of changes needed to get it working? 99% of this is unrelated

Reply:
Done